winres/winres.json

```diff
@@ -0,0 +1,52 @@
+{
+  "RT_GROUP_ICON": {
+    "OTHER": {
+      "0000": "hazelcast-icon-flat-sm.png"
+    }
+  },
+  "RT_MANIFEST": {
+    "#1": {
+      "0409": {
+        "identity": {
```

Comment:
In the docs it says:

> It is recommended to omit identity if your program is a plain application, not meant to be a side-by-side dependency.

So maybe we can leave that out?

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/118/commits/3133e5e22d274ecc42ad27854d72998c1985632e